### DISCUSSION TITLE
Gopro HDMI output issue

### DISCUSSION THREAD

**Discussion by nicomsprod** [gopro/labs#312] [Feature Requests]
Hello
On gopro 9 for a  live stream i search to have the back screen of the gopro running during HDMi output...
someone have a Gopro lab code for that?
I have already fin the clean live feed with gopro lab but my screen of gopro turn off when hdmi is connected

**Comment by dnewman-gpsw** [comment]
There is not support for that, and unfortunately I don't believe that will be possible.  Design wise, it is one or the other.

**Comment by cprima** [comment]
@nicomsprod You might want to see my answer to the related question https://github.com/gopro/labs/discussions/315

Yes, it is a somewhat expensive answer of about $1000. But in the big picture of digital media production I believe I sketched out the least expensive answer.
Your usecase could fit the $300 ATEM mini (which only does 1 source monitor out, not "multiview", but that would match your answer. You would also gain a bunch of other features that a livestream benefits from).
